sig-windows/annual-report-2021.md

```diff
@@ -3,120 +3,82 @@
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+     - (1.22) [1122 - windows-csi-support](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1122-windows-csi-support/README.md)
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - (1.23) [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md)
+     - (1.23) https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission/kep.yaml
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - (1.22) [1981 - Windows Privileged Container Support](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md)
+     - (1.23) [2802 -Identify Windows pods at API admission level authoritatively](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2802-identify-windows-pods-apiserver-admission/README.md)
+   - Pre-alpha (Targeting 1.24)
+     - [2578 - Windows Conformance](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/2578-windows-conformance/kep.yaml)
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
-
-   -
-   -
-   -
-
+   - csi-proxy and storage: this seems like an underserved area for windows https://github.com/kubernetes-csi/csi-proxy (meeting info is there).
 2. What metrics/community health stats does your group care about and/or measure?
-
-   -
-   -
-   -
-
+   - stars for ksigs/windows specific repos
+     - sig-windows-dev-tools   
+       - https://github.com/kubernetes-sigs/sig-windows-dev-tools -> up to 46, represents interest
+     - sig-windows-tools
+       - represents people trying to install windows on k8s nodes
+     - windows-gmsa
+       - represents enterprises integrating windows pods into GMSA
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
-
-   -
-
+   - yes
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
-
-   -
+   - yes
```

Comment:
still not seeing this... if this isn't there yet, describe the gap and link to an issue tracking resolving it?

Reply:
I recently updated the contributing guide with #6433

https://github.com/kubernetes/community/blob/master/sig-windows/CONTRIBUTING.md#creating-a-pr mentions to follow the general contributing guide but add `/sig windows` and trigger additional e2e PRs

PTAL

Reply:
thanks